<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlyEmOrthoControlForm</class>
 <widget class="QWidget" name="FlyEmOrthoControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>261</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="locateToPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Locate in Main Window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetCrosshairPushButton">
       <property name="text">
        <string>Reset Crosshair</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="dataCheckBox">
       <property name="text">
        <string>Data</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showSegCheckBox">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../gui.qrc">
         <normaloff>:/images/view_segmentation.png</normaloff>:/images/view_segmentation.png</iconset>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="contrastCheckBox">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../gui.qrc">
         <normaloff>:/images/bc_enhance.png</normaloff>:/images/bc_enhance.png</iconset>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="smoothCheckBox">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../gui.qrc">
         <normaloff>:/images/smooth.png</normaloff>:/images/smooth.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="crosshairCheckBox">
       <property name="text">
        <string>Crosshair</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ZFlyEmMessageWidget" name="messageWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZFlyEmMessageWidget</class>
   <extends>QWidget</extends>
   <header>flyem/zflyemmessagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gui.qrc"/>
 </resources>
 <connections/>
</ui>
